about.html

```diff
@@ -0,0 +1,126 @@
+    <link rel="stylesheet" href="https://use.typekit.net/vrw5nhb.css">
+    <script src="https://kit.fontawesome.com/5706a27a8e.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section class="mobileMenu hide">
+        <div class="menuContainer">
+            <ul>                
+                <li> <a href="index.html">Home</a> </li>
+                <li> <a href="about.html"class="about">About</a> </li>
+                <li> <a href="index.html#program">Program</a> </li>
+                <li> <a href="#">Join</a> </li>
+                <li> <a href="#">Sponsor</a> </li>
+                <li> <a href="#">News</a> </li>                
+            </ul>
+        </div>
+    </section>
+   <section>
+       <section class="deskSection">
+           <div class="topNavContainer">
+               <ul class="topNavContainerUl">
+                   <li><a href="#"><img src="img/github.png" alt="github logo"></a></li>
+                   <li><a href="#"><img src="img/twitter.png" alt="twitter logo"></a></li>
+                   <li><a href="">Logout</a></li>
+                   <li><a href="">Mypage</a></li>
+                   <li><a href="">English</a></li>
+               </ul>
+           </div>
+           <nav class="deskSection">
+               <div class="topBarContainer">
+                   <a href="index.html">
+                       <div class="logo">
+                           <p>Techsphere</p>
+                           <i class="fa fa-globe fa-3x" aria-hidden="true"></i>
+                       </div>
+                   </a>
+                   <ul class="topBarUl">
+                       <li><a id="about" href="about.html">About</a></li>
+                       <li><a href="index.html#">Program</a></li>
+                       <li><a href="#">Join</a></li>
+                       <li><a href="#">Sponsor</a></li>
+                       <li><a href="#">News</a></li>
+                   </ul>
+                   <button class="topBarBtn">Campaign</button>
+               </div>
+           </nav>
+       </section>
+       <div class="aboutIntroContainer">
+           <nav class="aboutNav mobile popup">
+               <span class="rect rec1"> </span>
+               <span class="rect rec2"> </span>
+               <span class="rect rec3"> </span>
+           </nav>           
+           <p class="aboutHello">"Hello World!"</p>
+           <h1 class="aboutIntroH1">Techsphere Global <br>Summit 2022</h1>
+           <p class="aboutTechSphere">The Techsphere Global Summit brings together the community of experts, academics, activists and enthusiasts who comprise the bustling tech space. This year, we hope to expand our invitation list including organizations and individuals</p>
+           <p class="contactInfo">Please contact us via Email for any further enquiries about Techsphere Global Summit 2022!</p>
+           <a class="contactEmail" href="#">techspheresubmit22@gmail.com</a>
+       </div>
```

Comment:
- [ ] [REQUIRED] Kindly fix the changes in the image below:

| Your Design |
|--------------|
|![image](https://user-images.githubusercontent.com/62938735/190866422-a1b750af-3017-4f9c-848e-e8358a1bba9f.png)|